.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "eslint:recommended",
+  "parserOptions": {
+    "ecmaVersion": 8    
```

Comment:
Setting this to 8 will result in some features __not__ supported by Node 6 to no longer be caught by ESLint, so we should beware of this.

Reply:
Actually, babel seem used everywhere in parcel. It allows to use `async`/`await` for instance, which is not supported by Node 6 and si part of ecmaScript 2017 or 8. That's why this version here is needed.

Reply:
Yup, you're right. The whole codebase is later transpiled using Babel. This should be okay. Thanks for the clarification 😅